sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>This is the legacy Azure Event Hubs .NET Standard Event Processor Host library. To view releases for the current generation of the Azure client libraries for .NET, please see: https://aka.ms/azsdk/releases.  For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
-    <Version>4.4.0-beta.1</Version>
+    <Description>Please note, a newer package Azure.Messaging.EventHubs.Processor is available as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```

Comment:
```suggestion
    <Description>Please be aware that a newer package, Azure.Messaging.EventHubs.Processor, was released in February 2020.  While this package will continue to receive critical bug fixes, we strongly encourage upgrading for new development. Read the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```

Reply:
In my opinion, the `for new development` is a significant distinction.  While we'd like to see migration in existing applications, I don't believe there is forcing function for it nor that there is enough of a value proposition to justify the efforts for many applications with an existing investment in the legacy packages.

Reply:
Keeping the current text as concluded in https://github.com/Azure/azure-sdk-for-net/pull/20402#discussion_r614228896